src/workers/fetchers/BaseFetcher.ts

```diff
@@ -26,6 +27,20 @@ export abstract class BaseFetcher<T extends WorkerConfiguration.FetcherConfigura
     }
   }
 
+  override async init() {
```

Comment:
We have to define a init function because the constructor does not allow async functions and "getWorkersConfiguration" is async